From 7577a63e148934721cd88a485dbfe51180fef079 Mon Sep 17 00:00:00 2001
From: Ludovic Barre <ludovic.barre@foss.st.com>
Date: Tue, 24 Sep 2024 17:22:56 +0200
Subject: [PATCH 3/3] app: fix a35 build error on cpu interface

tf-m-tests/app/main_ns.c: In function 'tfm_ns_start_copro':
tf-m-tests/app/main_ns.c:155:29: error: storage size of 'cpu_info' isn't known

Signed-off-by: Ludovic Barre <ludovic.barre@foss.st.com>
Change-Id: Id89654ed3c60474eee3fc3fb77cc359ffbec3008
---
 app/main_ns.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/main_ns.c b/app/main_ns.c
index 7884559..12adeda 100644
--- a/app/main_ns.c
+++ b/app/main_ns.c
@@ -142,6 +142,7 @@ __WEAK int32_t tfm_ns_cp_init(void)
     return ARM_DRIVER_OK;
 }
 
+#ifdef STM32_M33TDCID
 __WEAK const char *cpu_status_str[] = {
 	"offline",
 	"suspended",
@@ -181,7 +182,6 @@ void tfm_ns_start_copro(void *argument)
 	LOG_MSG("done\r\n");
 }
 
-#ifdef STM32_M33TDCID
 static osThreadFunc_t ca35_thread_func = tfm_ns_start_copro;
 static const osThreadAttr_t ca35_thread_attr = {
     .name = "CA35_thread",
-- 
2.34.1

